tests/chainer_tests/functions_tests/math_tests/test_minmax.py

```diff
@@ -13,7 +13,7 @@
 class TestMax(unittest.TestCase):
 
     def setUp(self):
-        eps = 1e-5
+        eps = 1e-6
```

Comment:
This makes the test harder.

Reply:
Sorry, my previous comment wasn't true.

- The precondition `(self.x > (numpy.max(self.x) - 2 * eps)).sum() == 1` becomes weaker.
- The backward check with `'eps': eps` might be more accurate.